app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,26 @@
+post "/questions/:survey_id" do
+  question = Question.new(body: params[:question])
+  @survey = Survey.find(params[:survey_id])
+  if question.save
+    @survey.questions << question
+    question.choices << Choice.create(body: params[:choice1])
+    question.choices << Choice.create(body: params[:choice2])
+    question.choices << Choice.create(body: params[:choice3])
+    question.choices << Choice.create(body: params[:choice4])
```

Comment:
Check out how the other groups are doing this with `name="choices[]"` in the view and getting the params as an array...